
---
bool frontierExists
geometry_msgs/Point frontierPoint